modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/installing-vsphere-installer-provisioned.adoc
+
+[id="installation-vsphere-installer-infra-requirements_{context}"]
+= vCenter requirements
+
+Before you install a {product-title} cluster on your vCenter that uses infrastructure that the installer provisions, you must prepare your environment.
+
+[discrete]
+= Required vCenter account privileges
+
+To install an {product-title} cluster in a vCenter, the installation program requires access to an account with privileges to read and create the required resources. Using an account that has administrative privileges is the simplest way to access all of the necessary permissions.
+
+A user requires the following privileges to install an {product-title} cluster:
+
+* Datastore
+** *Allocate space*
+* Folder
+** *Create folder*
+** *Delete folder*
+* vSphere Tagging
+** All privileges
+* Network
+** *Assign network*
+* Resource
+** *Assign virtual machine to resource pool*
+* Profile-driven storage
+** All privileges
+* vApp
+** All privileges
+* Virtual machine
+** All privileges
+
+For more information about creating an account with only the required privileges, see link: https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.security.doc/GUID-5372F580-5C23-4E9C-8A4E-EF1B4DD9033E.html[vSphere Permissions and User Management Tasks] in the vSphere documentation.
+
+[discrete]
+== Cluster resources
+
+When you deploy an {product-title} cluster that uses installer-provisioned infrastructure, the installation program must be able to create several resources in your vCenter instance.
+
+A standard {product-title} installation creates the following vCenter resources:
+
+* 1 Folder
+* 1 Tag category
+* 1 Tag
+* Virtual machines:
+** 1 template
+** 1 temporary bootstrap node
+** 3 control plane nodes
+** 3 compute machines
+
+These resources require a minimum of 800 GB of storage.
```

Comment:
The default storage for each node is 120G, plus 1 template with 16G, so the minimum size of storage required is 856 GB.

Reply:
this is technically correct. for vanilla ipi install bootstrap node will be torn down before worker creation so 800 gb is safe. I am fine with bumping as suggested.